src/kubernetes/nestjs-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nestjs-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nestjs
+  template:
+    metadata:
+      labels:
+        app: nestjs
+    spec:
+      initContainers:
+        - name: migration
+          image: suprugafad/medium-clone:latest
+          command: [ "npm", "run", "db:migrate" ]
+          env:
+            - name: DATABASE_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: pg-secrets
+                  key: POSTGRES_USER
+            - name: DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: pg-secrets
+                  key: POSTGRES_PASSWORD
+          envFrom:
+            - configMapRef:
+                name: nestjs-configmap
+      containers:
+        - name: nestjs
+          image: suprugafad/medium-clone:latest
```

Comment:
@suprugafad latest image version is not a good idea

Reply:
fixed